openapi/components/schemas/Common/CommonTransactionRequest.yaml

```
@@ -14,8 +14,7 @@ properties:
     allOf:
       - $ref: ../ResourceId.yaml
   currency:
-    allOf:
-      - $ref: ../CurrencyCode.yaml
+    - $ref: ../CurrencyCode.yaml
```

As far as I know `-` is not needed

```suggestion
    $ref: ../CurrencyCode.yaml
```